### DISCUSSION TITLE
KeePassXC Secret Service

### DISCUSSION THREAD

**Discussion by DominiqueFrt** [keepassxreboot/keepassxc#6158] [Q&A]
Bonjour,

Est-il envisageable de :

	1	Fermer l’interface graphique tout en maintenant ouverte l’application, tel que “Synology Drive Client”, “Dropbox”,          “Sirikali” ou “Vorta”.
	2	Lancer une application utilisant “libsecret” tel que “Sirikali” ou “Vorta” après le dévérouillage de la base de donnée

Cordialement
________________________________________________________________________________________________
Hi,

Is it possible to:

1 	Close the GUI while keeping the application open, such as “Synology Drive Client”, “Dropbox”, “Sirikali” or “Vorta”.
2 	Launch an application using “libsecret” such as “Sirikali” or “Vorta” after unlocking the database.

[Screenshot_20210220_123910.pdf](https://github.com/keepassxreboot/keepassxc/files/6014931/Screenshot_20210220_123910.pdf)
 
Cordially

**Comment by phoerious** [comment]
Hi, simply go to settings, enable the tray icon and check "Minimise instead of app exit".

I am not sure I understand your second question. KeePassXC can work as a secret service via libsecret.

**Comment by DominiqueFrt** [comment]
Hi,
See attached file
Cordially

**Reply by droidmonkey** [reply]
Please don't attach pdf files, we won't open them. Attach images only.

**Comment by DominiqueFrt** [comment]
![part-1](https://user-images.githubusercontent.com/42612895/109537387-1daab100-7abf-11eb-871c-13138806dc72.png)
![part-2](https://user-images.githubusercontent.com/42612895/109537418-256a5580-7abf-11eb-9363-318aa0daa99a.png)
![part-3](https://user-images.githubusercontent.com/42612895/109537451-2e5b2700-7abf-11eb-92fa-277409ff526b.png)
